### DISCUSSION TITLE
Erlang_LS extension will not start separte servers for two remote projects.

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#21885] [Feature Requests]
When an Erlang project is opened, the erlang-ls extension starts and works as usual. If a second project is opened, while keeping the previous one opened as well, erlang-ls for the second project will not start. This happens for remote projects.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
